core/doc_data.h

```diff
@@ -116,6 +116,17 @@ class DocData {
 		}
 	};
 
+	struct ThemeItemDoc {
+		String name;
+		String type;
+		String data_type;
+		String description;
+		String default_value;
+		bool operator<(const ThemeItemDoc &p_prop) const {
```

Comment:
If we're pedantic this should be `p_theme_item` or similar :)
But it's not a big deal and those are also "theme properties" so one can argue the name is fine as is.

Reply:
Are we being pedantic? 🙃